wemake_python_styleguide/violations/refactoring.py

```diff
@@ -1222,3 +1224,56 @@ class ImplicitNegativeIndexViolation(ASTViolation):
 
     error_template = 'Found implicit negative index'
     code = 530
+
+
+@final
+class SimplifiableReturningIfStatementViolation(ASTViolation):
+    """
+    Forbid if statements that simply return booleans in functions or methods.
+
+    Reasoning:
+        There is no need to test a condition and simply return a boolean
+        depending on its outcome if there is not going to be any additional
+        code.
+
+    Solution:
+        Instead of testing the condition and returning a boolean, return the
+        condition itself.
+
+    Example::
+
+        # Correct:
+        def some_function():
```

Comment:
Just one correct and just one wrong examples are fine.

Reply:
I agree on leaving one wrong example but from my experience using this project at work, I'd prefer to leave the two first correct examples so as to avoid confusion because those two examples are pretty different from one another and still correct. I'll leave only one if you insist though.